packages/core/saved-objects/core-saved-objects-api-browser/src/apis/create.ts

```diff
@@ -35,4 +35,6 @@ export interface SavedObjectsCreateOptions {
   typeMigrationVersion?: string;
   /** Array of referenced saved objects. */
   references?: SavedObjectReference[];
+  /** Boolean flag to indicate a document is managed, used during import to create/bulkCreate saved objects. */
```

Comment:
```suggestion
/**
   * Flag indicating if a saved object is managed by Kibana (default=false)
   *
   * This can be leveraged by applications to e.g. prevent edits to a managed
   * saved object. Instead, users can be guided to create a copy first and
   * make their edits to the copy.
   */
```